pages.fr/common/bundle.md

```diff
@@ -0,0 +1,36 @@
+# bundle
+
+> Gestionnaire de dépendances pour le langage de programmation Ruby.
+> Plus d'informations : <https://bundler.io/man/bundle.1.html>.
+
+- Installe toutes les gems définies dans le `Gemfile` attendu dans le répertoire de travail :
+
+`bundle install`
+
+- Exécute une commande dans le contexte du bundle actuel :
+
+`bundle exec {{commande}} {{arguments}}`
+
+- Met à jour toutes les gems selon les règles définies dans le `Gemfile` et régénérer `Gemfile.lock` :
```

Comment:
```suggestion
- Mets à jour toutes les gems selon les règles définies dans le `Gemfile` et régénére le fichier `Gemfile.lock` :
```
Can you please use impérative mood please ? :D 

Also you can conjugate the verb "regenerate"